scanners/zap/Makefile

```diff
@@ -10,4 +10,15 @@ scanner = zap
 
 include ../../scanners.mk
 
-deploy-test-deps: deploy-test-dep-nginx
+deploy-test-deps: deploy-test-dep-juiceshop deploy-test-dep-nginx deploy-test-dep-bodgeit deploy-test-dep-petstore
+
+#Run integration tests for the ZAP Automation Framework Scanner.
+integration-tests:
+	@echo ".: 🩺 Starting integration test in kind namespace 'integration-tests'."
+	kubectl -n integration-tests delete scans --all
+	kubectl -n integration-tests delete configmaps --all
+	helm -n integration-tests upgrade --install $(scanner) ./ --wait \
+		--set="parser.image.repository=docker.io/$(IMG_NS)/$(parser-prefix)-zap" \
+		--set="parser.image.tag=3.1.0"
```

Comment:
Why is the image tag fixed to a specific version here and why do we need to override the general make target `integration-test`? If there is a good reason there should be an explanation in the comment...